ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/BallerinaShell.java

```diff
@@ -235,4 +265,79 @@ public void reset() {
             throw new RuntimeException(e);
         }
     }
+
+    /**
+     * Add modules exists in the Ballerina Distribution repo.
+     *
+     * @param moduleImporter moduleImporter.
+     * @param modules        available modules.
+     */
+    public void importModules(ModuleImporter moduleImporter, List<String> modules) {
+        List<String> missingModules = new ArrayList<>();
+        List<String> requiredModules = new ArrayList<>();
+        terminal.info("Found following undefined module(s).");
+        for (String module : modules) {
+            module = module.replaceAll("'", "");
```

Comment:
If you are trying to replace the single quote here, let's use `replace` instead of `replaceAll` since the latter accepts a regex.

Reply:
Updated in 9f270db8b7259fb63e77545e8454623db4d6c3a7